core/snmp/proxy-rpc-impl/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -12,6 +12,8 @@
 ">
 
     <service interface="org.opennms.core.rpc.api.RpcModule" >
-        <bean class="org.opennms.netmgt.snmp.proxy.common.SnmpProxyRpcModule" />
+        <bean class="org.opennms.netmgt.snmp.proxy.common.SnmpProxyRpcModule">
+            <property name="secureCredentialsVault" ref="dominionGrpcSecureCredentialsVault" />
```

Comment:
There are multiple versions of Secure Credentials Vault on Minion but ideally only one version may exist and that may not be always `dominionGrpcSecureCredentialsVault`.

So take reference of the available interface.
    `<reference id="secureCredentialsVault" interface="org.opennms.features.scv.api.SecureCredentialsVault" availability="mandatory"/>`